Misha/Text/OB-179_count_vowels/count_vowels_main.py

```diff
@@ -0,0 +1,20 @@
+def count_vowels(text):
```

Comment:
Can you add function that will count overall value of vowels? 
It will get {'a': 2, 'e': 5, 'i': 6, 'o': 4, 'u': 2} and return 19